content/events/2021-online-unconference/_index.md

```diff
@@ -26,26 +26,30 @@ who are or identify as RSEs, but anybody interested in the interface of research
 
 ## What contributions we welcome
 
-In principle we welcome any contributions which respect the [Code of Conduct](./code-of-conduct)
-and in principle of any length. We will try to arrange a program based on your input.
+We welcome *any kind of contribution* which respects the [Code of Conduct](./code-of-conduct). 
+We will try to arrange a program based on your input.
 
 Here are some examples of possible contributions:
-- Discussion topics (about software, the research software community, or anything related to RSEs)
-- Demonstrations (tools you like, software you wrote)
-- Quick workshops or [ReproHacks](https://reprohack.github.io/reprohack-hq/) (only 3 hours in a day)
-- Talks 
-- Workshops and lessons: teach us something new
-- Can also be something unfinished and almost working where you would like to get feedback on
+- *Discussion topics*: is software, research software community or research software engineer something close to your heart? Come chatting with other enthusiasts!
+- *Demonstrations:* show us some tools or softwares that you like
+- *Workshop or [ReproHacks](https://reprohack.github.io/reprohack-hq/)* (only 3 hours in a day)
+- *Talks*: teach us something new 
+- *On going projects*: share with others something unfinished and almost working where you would like to get feedback on
+- *Something else*: something you would like to share with others that does not fit any of the previuous points? We are looking forward to hearing about it!
```

Comment:
if someone wants to share an "ongoing projects" or "something else" (although I guess this last one will be rare, since I think the list is quite exhaustive), what should they mark in the issue template, currently they are not listed there. I guess "ongoing project" can go under demonstration or talk, but should there be an "other" box in the template?

Reply:
This is also where our feedback/help comes in. :)

Reply:
I added "Something else" to the issue template